ajax/libs/primeng/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "primeng",
+  "filename": "primeng.min.js",
+  "description": "PrimeNG is a collection of rich UI components for Angular 2",
+  "version": "2.0.1",
+  "keywords": [
+    "angular",
+    "ui",
+    "components",
+    "datatable",
+    "datagrid",
+    "charts",
+    "typescript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/primefaces/primeng.git"
+  },
+  "homepage": "http://www.primefaces.org/primeng/",
+  "author": "cagataycivici",
+  "license": "MIT",
+  "npmName": "primeng",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "primeng.js*",
+        "resources/**/!(*.scss)"
```

Comment:
As the issue about what should we host you mentioned, I think we only need two themes (bootstrap and omega). Here you include so many other themes' files. Also, they are added in this commit, please remove them.